changelog.md

```diff
@@ -1,6 +1,12 @@
 # Backpack changelog
 
-_Nothing yet..._
+**Added:**
+- bpk-component-link:
+  - The `alternate` style can now be themed.
+
+**Changed:**
```

Comment:
`Changed` isn't a valid heading (we renamed them all to `Breaking`).

Also this isn't a breakign changed show it should be `Fixed`

Reply:
See this is why I don't like these names. Having a note about deprecation under a category called `Fixed` makes no sense 😶

Reply:
Well technically you "fixed" a dodgy api by exposing a newer, more sensible api ;)